docs/build.sh

```diff
@@ -18,5 +18,24 @@
 # specific language governing permissions and limitations
 # under the License.
 
-rm -r _build
-make html
+set -e
+
+FWDIR="$(cd "`dirname "$0"`"; pwd)"
+cd "$FWDIR"
+
+[ -d _build ] && rm -r _build
+
+NUM_IGNORED_WARNINGS=4
```

Comment:
why ignore warning is 4?

Reply:
https://github.com/apache/airflow/pull/4588#issuecomment-457779608
4 warnings have not been fixed. I wanted to prevent an increase in the number of warnings, so i prepared a PR. now there are new errors, so it is necessary to change this number, or to correct errors.